Tue Jan 28 2025 14:0:45 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:0:45 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:1:55 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:1:56 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:2:28 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:2:29 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:3:54 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:3:55 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:4:25 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:4:26 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:16:8 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:16:9 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:17:12 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:17:12 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:18:41 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:18:42 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:19:43 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:19:44 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:21:23 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:21:24 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:30:19 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:30:20 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:33:50 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:33:51 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:34:3 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:34:4 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:34:41 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:34:42 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:35:2 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:35:2 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:35:57 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:35:58 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:36:18 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:36:19 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:37:23 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:37:23 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:40:26 [Podlove] [31merror[39m: User validation failed: age: Path `age` (33) is less than minimum allowed value (35).
ValidationError: User validation failed: age: Path `age` (33) is less than minimum allowed value (35).
    at model.Document.invalidate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/document.js:3318:32)
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/document.js:3079:17
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1388:9
    at processTicksAndRejections (node:internal/process/task_queues:85:11)
ValidationError
Tue Jan 28 2025 14:41:6 [Podlove] [31merror[39m: User not found
NotFoundError: User not found
    at match (/home/mir/dev/projects/podlove_backend/src/services/matchesServices.ts:13:37)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
NotFoundError
Tue Jan 28 2025 14:41:50 [Podlove] [31merror[39m: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
CastError: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
    at SchemaObjectId.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schema/objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1236:12)
    at SchemaObjectId.SchemaType.castForQuery (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1653:17)
    at cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/cast.js:390:32)
    at model.Query.Query.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4906:12)
    at model.Query.Query._castConditions (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2309:10)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2632:8)
    at model.Query.exec (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4458:80)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async matchedUsers (/home/mir/dev/projects/podlove_backend/src/services/matchesServices.ts:34:25)
CastError
Tue Jan 28 2025 14:42:40 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:42:40 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:42:45 [Podlove] [31merror[39m: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
CastError: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
    at SchemaObjectId.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schema/objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1236:12)
    at SchemaObjectId.SchemaType.castForQuery (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1653:17)
    at cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/cast.js:390:32)
    at model.Query.Query.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4906:12)
    at model.Query.Query._castConditions (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2309:10)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2632:8)
    at model.Query.exec (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4458:80)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async matchedUsers (/home/mir/dev/projects/podlove_backend/src/services/matchesServices.ts:34:25)
CastError
Tue Jan 28 2025 14:42:57 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:42:58 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:43:1 [Podlove] [31merror[39m: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
CastError: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
    at SchemaObjectId.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schema/objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1236:12)
    at SchemaObjectId.SchemaType.castForQuery (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1653:17)
    at cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/cast.js:390:32)
    at model.Query.Query.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4906:12)
    at model.Query.Query._castConditions (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2309:10)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2632:8)
    at model.Query.exec (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4458:80)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async matchedUsers (/home/mir/dev/projects/podlove_backend/src/services/matchesServices.ts:34:25)
CastError
Tue Jan 28 2025 14:43:2 [Podlove] [31merror[39m: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
CastError: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
    at SchemaObjectId.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schema/objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1236:12)
    at SchemaObjectId.SchemaType.castForQuery (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1653:17)
    at cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/cast.js:390:32)
    at model.Query.Query.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4906:12)
    at model.Query.Query._castConditions (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2309:10)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2632:8)
    at model.Query.exec (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4458:80)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async matchedUsers (/home/mir/dev/projects/podlove_backend/src/services/matchesServices.ts:34:25)
CastError
Tue Jan 28 2025 14:43:3 [Podlove] [31merror[39m: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
CastError: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
    at SchemaObjectId.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schema/objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1236:12)
    at SchemaObjectId.SchemaType.castForQuery (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1653:17)
    at cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/cast.js:390:32)
    at model.Query.Query.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4906:12)
    at model.Query.Query._castConditions (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2309:10)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2632:8)
    at model.Query.exec (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4458:80)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async matchedUsers (/home/mir/dev/projects/podlove_backend/src/services/matchesServices.ts:34:25)
CastError
Tue Jan 28 2025 14:43:13 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:43:14 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:43:20 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:43:21 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:43:34 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:43:35 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:43:38 [Podlove] [31merror[39m: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
CastError: Cast to ObjectId failed for value ":id" (type string) at path "_id" for model "User"
    at SchemaObjectId.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schema/objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1236:12)
    at SchemaObjectId.SchemaType.castForQuery (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/schemaType.js:1653:17)
    at cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/cast.js:390:32)
    at model.Query.Query.cast (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4906:12)
    at model.Query.Query._castConditions (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2309:10)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2632:8)
    at model.Query.exec (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:4458:80)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async matchedUsers (/home/mir/dev/projects/podlove_backend/src/services/matchesServices.ts:34:25)
CastError
Tue Jan 28 2025 14:43:57 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:43:58 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:44:1 [Podlove] [31merror[39m: Cannot populate path `first` because it is not in your schema. Set the `strictPopulate` option to false to override.
StrictPopulateError: Cannot populate path `first` because it is not in your schema. Set the `strictPopulate` option to false to override.
    at getModelsMapForPopulate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/helpers/populate/getModelsMapForPopulate.js:50:12)
    at populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4323:21)
    at _populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4283:5)
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4255:5
    at new Promise (<anonymous>)
    at Function.populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4254:10)
    at model.Query.Query._completeOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2575:18)
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2647:10
    at new Promise (<anonymous>)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2646:10)
StrictPopulateError
Tue Jan 28 2025 14:44:56 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:44:57 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:45:1 [Podlove] [32minfo[39m: Successfully Connected to MongoDB Atlas
Tue Jan 28 2025 14:45:2 [Podlove] [32minfo[39m: Server is running at PORT: 7000
Tue Jan 28 2025 14:45:5 [Podlove] [31merror[39m: Cannot populate path `first` because it is not in your schema. Set the `strictPopulate` option to false to override.
StrictPopulateError: Cannot populate path `first` because it is not in your schema. Set the `strictPopulate` option to false to override.
    at getModelsMapForPopulate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/helpers/populate/getModelsMapForPopulate.js:50:12)
    at populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4323:21)
    at _populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4283:5)
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4255:5
    at new Promise (<anonymous>)
    at Function.populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4254:10)
    at model.Query.Query._completeOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2575:18)
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2647:10
    at new Promise (<anonymous>)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2646:10)
StrictPopulateError
Tue Jan 28 2025 14:45:13 [Podlove] [31merror[39m: Cannot populate path `first` because it is not in your schema. Set the `strictPopulate` option to false to override.
StrictPopulateError: Cannot populate path `first` because it is not in your schema. Set the `strictPopulate` option to false to override.
    at getModelsMapForPopulate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/helpers/populate/getModelsMapForPopulate.js:50:12)
    at populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4323:21)
    at _populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4283:5)
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4255:5
    at new Promise (<anonymous>)
    at Function.populate (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/model.js:4254:10)
    at model.Query.Query._completeOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2575:18)
    at /home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2647:10
    at new Promise (<anonymous>)
    at model.Query._findOne (/home/mir/dev/projects/podlove_backend/node_modules/.pnpm/mongoose@8.8.4/node_modules/mongoose/lib/query.js:2646:10)
StrictPopulateError
